### PR TITLE
Add rem books

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review this PR.

Task: A basic website that allows users to add/remove books from a list

Changes:
- Add index,html file with the book title, and author inputs
- Use JavaScript objects and arrays to store books
- Add books, remove books, and display all the books
- Store and retrieve books from localStorage
- Dynamically modify the DOM and add basic events
- Add Linters, README, LICENSE

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const bookAuthor = document.createElement('h4');
+    bookAuthor.textContent = myBook.bookAuthor;
+
+    // Create remove button
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+
+    // Add event listener to remove button
+    removeBtn.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
```
Comment (mwanawabangona) [inline script.js]: - [x] `required`
- [x] Please enable the disabled `eslint` because it defeats the purpose of adding linters. 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @egichuhis ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (egichuhis) [reply script.js]: Hello @mwanawabangona , I need your insight here, if I enable the requested change, I will get a linter error, 
`'removeBook' was used before it was defined.eslint`. This does not affect the program execution at all. How should I go about this? Kindly advise. I have tried refactoring my code by re-organising the functions but still run into the same problem.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @egichuhis and @MohammadYaser  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes Implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**